common/app/services/DynamoDB.scala

```diff
@@ -42,12 +42,12 @@ class DynamoDB(wsClient: WSClient) extends Logging with ExecutionContexts {
       |{
       |   "TableName": "$tableName",
       |   "Key": {
-      |     "source": {"S": "$source"}
+      |     "source": {"S": "$expectedSourceProtocol$source"}
```

Comment:
WIP until later PR, but better the protocol is defined here than hidden in `ArchiveController` I thought